src/main/resources/javascript/locales/en.json

```diff
@@ -2,9 +2,11 @@
   "label": {
     "action": "Copy to other languages",
     "dialogTitle": "Copy “{{propertyName}}” to other languages",
+    "dialogTitleAllProperties": "Copy all properties to other languages",
     "languagesSelected": "{{count}} language selected",
     "languagesSelected_plural": "{{count}} languages selected",
     "dialogDescription": "The value of this field will be copied to all other languages, overwriting the current values.",
+    "dialogDescriptionAllProperties": "The values of all properties will be copied to all other languages, overwriting the current values.",
```

Comment:
Doesn't seem to be matching the text from story description. Do we need to update story?

Reply:
good catch, I missed those